importer/UaUk.py

```diff
@@ -0,0 +1,110 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class UaUk(Monument):
+
+    def set_image(self):
+        """
+        Set image if it's actually one.
+
+        Ignore if image is template saying there
+        will be no image because of freedom of panorama
+        issues.
+        """
+        if self.has_non_empty_attribute("image"):
+            if self.image == '{{неволя}}':
+                super().set_image
```

Comment:
So if it is a template then we add it as an image? shouldn't it be `!=`

Also should this not be `set_image()`